src/components/AnimatedCheckmark.vue

```diff
@@ -58,29 +55,15 @@ const props = defineProps({
   /**
    * The color of the checkmark.
    *
-   * Valid colors are:
-   * - Available CSS-colors (for example: white, #000, etc.)
-   * - Any custom CSS-class (a-zA-Z_-).
-   *
-   * If the color is an CSS-class, this class has to set the following properties:
-   * - fill: [color]
-   * - stroke: [color]
-   *
-   * If not set, it will be white.
+   * Defaults to white.
    */
   color: { type: String, default: "white" },
```

Comment:
I think we should change the default color to black or primary since its always confusing if you use it and it is white. Or no default color and the color is required.

Reply:
We can discuss this, but I think that is for another PR. Out of scope for this one.